docs/source/distributed.rst

```diff
@@ -787,3 +787,24 @@ With the ``NCCL`` backend, such an application would likely result in a hang whi
 In addition, `TORCH_DISTRIBUTED_DEBUG=DETAIL` can be used in conjunction with `TORCH_SHOW_CPP_STACKTRACES=1` to log the entire callstack when a collective desynchronization is detected. These
 collective desynchronization checks will work for all applications that use ``c10d`` collective calls backed by process groups created with the
 :func:`torch.distributed.init_process_group` and :func:`torch.distributed.new_group` APIs.
+
+Logging
```

Comment:
(can be done in followup PRs) does it make sense if we move `NCCL_DEBUG` and `NCCL_DEBUG_SUBSYS` to this section as well?

Reply:
Yes, ideally we should document all environment variables that are relevant to torch.distributed. Do you mind opening an Issue and listing all variables that you can think of? This was also explicitly asked by HG folks.